plugins/auth-backend/src/providers/providers.ts

```diff
@@ -83,4 +85,5 @@ export const defaultAuthProviderFactories: {
   bitbucket: bitbucket.create(),
   bitbucketServer: bitbucketServer.create(),
   atlassian: atlassian.create(),
+  guest: guest.create(),
```

Comment:
We're avoiding introducing any new built-in providers in the auth backend. It's worth a discussion whether this one is special, but for now I'd say it's best to avoid

Reply:
If my proposed changes to the `guestProvider` are the in the right direction, this might make sense to add to the old backend? At least until folks get transferred over to the new backend.